src/vs/workbench/contrib/terminal/common/terminal.ts

```diff
@@ -513,6 +513,7 @@ export const enum TerminalCommandId {
 	Split = 'workbench.action.terminal.split',
 	SplitInstance = 'workbench.action.terminal.splitInstance',
 	SplitInActiveWorkspace = 'workbench.action.terminal.splitInActiveWorkspace',
+	ShowQuickFixes = 'workbench.action.terminal.showQuickFixes',
```

Comment:
This should be `workbench.action.terminal.quickFix` like it was before? That aligns with the editor command's wording too

Reply:
I think that it should be a verb to align with all other commands

Reply:
I'm convinced 🙂